modules/platform/nuxeo-platform-directory/nuxeo-platform-directory-core/src/main/java/org/nuxeo/ecm/directory/DefaultDirectoryFactory.java

```diff
@@ -29,27 +28,21 @@
  * {@link DirectoryService}.
  *
  * @since 8.2
+ * @deprecated since 11.5: use a custom registry extending {@link AbstractDirectoryDescriptorRegistry} instead.
  */
+@Deprecated(since = "11.5")
 public class DefaultDirectoryFactory extends DefaultComponent {
```

Comment:
Can't we just remove this class? Given that the implementation now throws exceptions...

Reply:
We could indeed, I'm just not sure how to help people migrating (in case there are other directory implementations out there) but maybe that's not helping anyway.